src/actor/Verifier.spec.ts

```diff
@@ -207,6 +239,117 @@ describe('Verifier', () => {
     expect(owner).toBeDefined()
     expect((claims[0] as IRequestForAttestation).claim).toEqual(unownedClaim)
   })
+  it('should throw ... during privacy enhanced presentation', async () => {
```

Comment:
Note: I kept all throws in the same test to avoid re-running the "setup". Also, most tests are basically the same with permutated input.